openpype/hosts/max/api/lib.py

```diff
@@ -150,10 +150,10 @@ def set_framerange(start_frame, end_frame):
     Todo:
         Current type is hard-coded, there should be a custom setting for this.
     """
-    rt.rendTimeType = 4
+    rt.rendTimeType = 3
     if start_frame is not None and end_frame is not None:
-        frame_range = "{0}-{1}".format(start_frame, end_frame)
-        rt.rendPickupFrames = frame_range
+        rt.rendStart = int(start_frame)
+        rt.rendEnd = int(end_frame)
 
 
 def get_multipass_setting(project_setting=None):
```

Comment:
Not related to this PR: This particular function appears to only be used in `submit_max_deadline`. Seems a bit off that this is in `lib`?

Reply:
the original function is to set the frame range before the deadline submission. it is already there before the PR...

Reply:
I understand it was already there - I'm just saying the the function name and its docstring are misleading. `set_render_frame_range` might make it more obvious.

Reply:
I think your reply is regarding the other comment. This comment I made is about line 159 for function `get_multipass_setting`. It's definitely pre-existing code from outside of this PR like I mentioned. It just seems weird to be in the lib since it's just a dictionary lookup and used in once place only.

Reply:
> I think your reply is regarding the other comment. This comment I made is about line 159 for function `get_multipass_setting`. It's definitely pre-existing code from outside of this PR like I mentioned. It just seems weird to be in the lib since it's just a dictionary lookup and used in once place only.

Yeah, sorry for the wrong spot!
yes it should be moved to somewhere else, but it is not quite related to this PR.